cmd/gateway/mirror/sample.yaml

```
@@ -0,0 +1,148 @@
+        connection_timeout: ""
+        header_table_size: 0
+        initial_conn_window_size: 0
+        initial_window_size: 0
+        interceptors: []
+        keepalive:
+          max_conn_age: ""
+          max_conn_age_grace: ""
+          max_conn_idle: ""
+          time: ""
+          timeout: ""
+        max_header_list_size: 0
+        max_receive_message_size: 0
+        max_send_message_size: 0
+        read_buffer_size: 0
+        write_buffer_size: 0
+      mode: GRPC
+      probe_wait_time: 3s
+      restart: true
+  health_check_servers:
+    - name: liveness
+      host: 0.0.0.0
+      port: 3000
+      http:
+        handler_timeout: ""
+        idle_timeout: ""
+        read_header_timeout: ""
+        read_timeout: ""
+        shutdown_duration: 5s
+        write_timeout: ""
+      mode: ""
+      probe_wait_time: 3s
+    - name: readiness
+      host: 0.0.0.0
+      port: 3001
+      http:
+        handler_timeout: ""
+        idle_timeout: ""
+        read_header_timeout: ""
+        read_timeout: ""
+        shutdown_duration: 0s
+        write_timeout: ""
+      mode: ""
+      probe_wait_time: 3s
+  metrics_servers:
+  startup_strategy:
+    - liveness
+    - grpc
+    - readiness
+  full_shutdown_duration: 600s
+  tls:
+    ca: /path/to/ca
+    cert: /path/to/cert
+    enabled: false
+    key: /path/to/key
+observability:
+  enabled: false
+  trace:
+    enabled: false
+gateway:
+  client:
+    addrs:
+      - vald-discoverer.default.svc.cluster.local:8081
+    health_check_duration: "1s"
+    connection_pool:
+      enable_dns_resolver: true
+      enable_rebalance: true
+      old_conn_close_duration: 3s
+      rebalance_duration: 30m
+      size: 3
+    backoff:
+      backoff_factor: 1.1
+      backoff_time_limit: 5s
+      enable_error_log: true
+      initial_duration: 5ms
+      jitter_limit: 100ms
+      maximum_duration: 5s
+      retry_count: 100
+    call_option:
+      max_recv_msg_size: 0
+      max_retry_rpc_buffer_size: 0
+      max_send_msg_size: 0
+      wait_for_ready: true
+    dial_option:
+      backoff_base_delay: 1s
+      backoff_jitter: 0.2
+      backoff_max_delay: 120s
+      backoff_multiplier: 1.6
+      enable_backoff: false
+      initial_connection_window_size: 0
+      initial_window_size: 0
+      insecure: true
+      keepalive:
+        permit_without_stream: false
+        time: ""
+        timeout: ""
+      max_msg_size: 0
+      min_connection_timeout: 20s
+      read_buffer_size: 0
+      tcp:
+        dialer:
+          dual_stack_enabled: true
+          keepalive: ""
+          timeout: ""
+        dns:
+          cache_enabled: true
+          cache_expiration: 1h
+          refresh_duration: 30m
+        tls:
+          ca: /path/to/ca
+          cert: /path/to/cert
+          enabled: false
+          key: /path/to/key
+      timeout: ""
+      write_buffer_size: 0
+    tls:
+      ca: /path/to/ca
+      cert: /path/to/cert
+      enabled: false
+      key: /path/to/key
```

```suggestion
      enabled: false
       ca: /path/to/ca
      cert: /path/to/cert
      key: /path/to/key
```
It would be more readable :smile: 
(Othsers' line are the same)